readthedocs/sso/models.py

```diff
@@ -53,6 +52,13 @@ class SSOIntegration(models.Model):
         blank=True,
     )
 
+    using_old_dashboard = models.BooleanField(
+        default=False,
```

Comment:
I think you need null=True and blank=True here; or... you need to run this migration _after_ deploy.